include/kernels/multi_reduce_core.cuh

```diff
@@ -15,16 +15,10 @@ namespace quda
 
     /**
        @brief Return the batch block size used for multi reductions.
-       For now, we leave this at 1 unless fast-compilation is enabled,
-       since it provides negligible benefit (but substantially longer
-       tuning).  When fast-compilation is enabled, the increased batch
-       count can dramatically improve performance.
+       For now, we leave this at 1, but this is something that will be
```

Comment:
Typo, should be "8"

Reply:
Actually that comment is completely wrong now since fast reduce compilation is gone now.  Will update in next push

Reply:
aa3f7b9db9b23ed8a60628df9b078e1d9076d216